Makefile

```diff
@@ -2,21 +2,30 @@
 PROTOCOL_PATH=$(GOPATH)/src/github.com/keybase/client/protocol
 AVDLC=$(PROTOCOL_PATH)/node_modules/.bin/avdlc
 
-test:
+test-static:
 	poetry run mypy pykeybasebot/
-	poetry run python -m pytest
 	poetry run flake8
 	poetry run isort -rc . --check-only
 	poetry run black . --check
 
+test-unit:
+	poetry run python -m pytest tests/unit -s
+
+test-accept:
+	poetry run python -m pytest tests/acceptance -s
```

Comment:
suggest having this naively test everything just in case someone accidentally adds a new folder and doesn't think to add it here

Reply:
What do you mean test everything? If we just test the `tests/` dir, don't we lose the distinction between acceptance/unit tests?

Reply:
the important things are (1) CI only testing unit tests, and (2) testing everything before pushing a new version. there's no harm in re-running the unit tests in `test-accept`, so i'd argue it's safer to test everything in `tests/` just to ensure that if someone creates a new top level folder (`tests/integration` or `tests/alex-just-wanna-verify-this-one-thing`), those tests aren't missed completely by both processes.